README.md

```diff
@@ -157,6 +157,13 @@ request method (e.g. `GET`) and run `rest.nvim`.
 4. Push to the branch (<kbd>git push origin my-new-feature</kbd>)
 5. Create a new Pull Request
 
+
+## Related software
+
+- [Hurl](https://hurl.dev/)
+- https://httpie.io/
```

Comment:
Did you forget to use the markdown link syntax here or was it intentionally? :p

Reply:
intentional, how do you suggest I present it ?